awscli/clidocs.py

```diff
@@ -179,6 +187,11 @@ def doc_option(self, arg_name, help_command, **kwargs):
         doc.style.dedent()
         doc.style.new_paragraph()
 
+    def doc_global_option(self, help_command, **kwargs):
+        doc = help_command.doc
+        doc.style.h2('Global Options')
+        doc.write_from_file(GLOBAL_OPTIONS_FILE)
+
```

Comment:
We should also make sure to include a changelog entry for this using the `scripts/new-change` script.